.github/pull_request_template.md

```diff
@@ -1,41 +1,56 @@
+<!-- Mention any related issue(s) here using appropriate keywords such
+<!-- as "closes" or "resolves" to auto-close them on merge. -->
 
-## 🧪 Testing
+## 🧪 Testing ##
 
-<!--- Please describe in detail how you tested your changes. -->
-<!--- Include details of your testing environment, and the tests you ran to -->
-<!--- see how your change affects other areas of the code, etc. -->
+<!-- How did you test your changes? How could someone else test this PR? -->
 
-## 📷 Screenshots (if appropriate)
+<!-- Include details of your testing environment, and the tests you ran to -->
+<!-- see how your change affects other areas of the code, etc. -->
 
-## 🚥 Types of Changes
+## 📷 Screenshots (if appropriate) ##
 
-<!--- What types of changes does your code introduce? -->
-<!--- Put an `x` in all the boxes that apply: -->
+<!-- Remove this section and header if not needed -->
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (causes existing functionality to change)
+## 🚥 Types of Changes ##
 
-## ✅ Checklist
+<!-- What types of changes does your code introduce? -->
+<!-- Add any relevant types and remove any that don't apply. -->
+<!-- Consider making a PR against the template in
+<!-- https://github.com/cisagov/.github for new types. -->
 
-<!--- Go over all the following points, and put an `x` in all the
-boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask.
-We're here to help! -->
+* Bug fix (non-breaking change which fixes an issue)
+* New feature (non-breaking change which adds functionality)
+* Breaking change (causes existing functionality to change)
+* Documentation change
+* Upstream update
+* Version bump
 
-- [ ] My code follows the code style of this project.
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
-- [ ] I have read the **CONTRIBUTING** document.
-- [ ] I have added tests to cover my changes.
-- [ ] All new and existing tests passed.
+## ✅ Checklist ##
+
+<!-- Remove any of the following that do not apply. -->
+<!-- Draft PRs should have one or more un-checked boxes. -->
+<!-- If you're unsure about any of these, don't hesitate to ask. -->
+<!-- We're here to help! -->
+
+* [ ] PR has an informative and human-readable title
+* [ ] Changes are limited to a single goal - _eschew scope creep!_
+* [ ] _All_ future TODOs are captured in issues, which are referenced
+      in code comments.
+* [ ] Assignee and relevant labels have been added
+* [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document
+* [ ] Code changes follow [cisagov code standards](https://github.com/cisagov/development-guide)
+* [ ] Repo and/or project documentation updated to reflect the changes in this PR
+* [ ] Tests have been added to cover the changes in this PR
+* [ ] All new and existing tests pass
```

Comment:
Some of these checkbox items have ending punctuation, and some do not.  (I am super-guilty of doing this.)
I suggest we normalize one way or the other.   I would prefer adding Full Stops (`U+002E` `.`).  https://www.compart.com/en/unicode/U+002E

Reply:
I put periods at the end of items that are obviously complete sentences and omit them otherwise.  I could change teh bullets so that all are complete sentences.

Please also comment on [the question above](https://github.com/cisagov/.github/pull/4#discussion_r518127363).

Reply:
See commit f151ecf.

Reply:
To be clear, I do not think we should add periods to the end of statements that are not complete sentences.

Reply:
Same here - periods yes on sentences, but no on fragments.

Reply:
I generally prefer the "no periods on bullets ever unless there are multiple sentences on one bullet" but I bow to y'all's historic preference for periods.

Reply:
> I generally prefer the "no periods on bullets ever unless there are multiple sentences on one bullet" but I bow to y'all's historic preference for periods.

I am OK with this rule too.

Reply:
Thanks for making these changes.  I'm really fine either way, just wanted to make sure it wasn't accidentally skipped.